.changeset/curly-bikes-train.md

```diff
@@ -0,0 +1,24 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+"__section": feature
+---
+Adds enablable allowed types to SchemaFactoryAlpha
+
+This adds the `enablable` API to [`SchemaFactoryAlpha`](https://fluidframework.com/docs/api/fluid-framework/schemafactoryalpha-class).
+Enablables can be used for schema evolution to add members to an [`AllowedTypes`](https://fluidframework.com/docs/api/fluid-framework/allowedtypes-typealias) while supporting cross version collaboration.
+
+Enablables are allowed types that can be read by the tree but not written. Attempting to write an enablable type will cause a runtime error.
+
+To add a new member to an `AllowedTypes`, add the type wrapped by `enablable`:
+```typescript
+schemaFactoryAlpha.arrayAlpha("TestArray", [schemaFactoryAlpha.number, schemaFactoryAlpha.enablable(schemaFactoryAlpha.string)]);
+```
+
+Once enough clients have this code update, it is safe to allow writing strings to the array.
+To enable writing strings to the array, a code change must be made to remove the enablable annotation:
+```typescript
+schemaFactoryAlpha.arrayAlpha("TestArray", [schemaFactoryAlpha.number, schemaFactoryAlpha.string]);
+```
+
+A future change will add an API that allows enablables to be enabled via a runtime schema upgrade so that the type can be enabled using a configuration flag change rather than a code change.
```

Comment:
Somone might read this thinking a future change refers to a future change of an app (like our example above). To clarify:

```suggestion
In the future, SharedTree may add an API that allows enablables to be enabled via a runtime schema upgrade so that the type can be more easily deployed using a configuration flag change rather than a code change.
```